### PR TITLE
 Data tokens are stored in plain text

### PR DESCRIPTION
**Pull request by nikitag-ms**
* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.

## Description

Data tokens are stored in plain text

### CODE REVIEW

File: sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/AbstractStorageTest.java [hunk 1]
```diff
@@ -206,14 +208,30 @@ public Object answer(InvocationOnMock invocation) {
...
 
         /* Mock utils. */
         mockStatic(CryptoUtils.class);
-        when(CryptoUtils.getInstance(any(Context.class))).thenReturn(mock(CryptoUtils.class));
         mockStatic(JSONUtils.class);
+
+        /* Mock CryptoUtils. */
+        CryptoUtils cryptoUtils = mock(CryptoUtils.class);
+        when(cryptoUtils.encrypt(anyString())).thenAnswer(new Answer<String>() {
+            @Override
```
Comment (guperrot) [inline sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/AbstractStorageTest.java]:
```suggestion
            
            @Override
```

File: sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/AbstractStorageTest.java [hunk 2]
```diff
@@ -206,14 +208,30 @@ public Object answer(InvocationOnMock invocation) {
...
+            public String answer(InvocationOnMock invocation) {
+                Object[] args = invocation.getArguments();
+                return (String) args[0];
+            }
+        });
+        when(cryptoUtils.decrypt(anyString(), anyBoolean())).thenAnswer(new Answer<CryptoUtils.DecryptedData>() {
+            @Override
```
Comment (guperrot) [inline sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/AbstractStorageTest.java]:
```suggestion
            
            @Override
```

File: sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java [hunk 3]
```diff
@@ -51,7 +54,22 @@
 
     private static final String READONLY_PARTITION_NAME = Constants.READONLY;
 
-    private static final String FAKE_TOKEN = "mock";
+    private static final TokenExchange.TokenExchangeServiceCallback TOKEN_EXCHANGE_SERVICE_CALLBACK =
+            new TokenExchange.TokenExchangeServiceCallback(TokenManager.getInstance(PowerMockito.mock(Context.class))) {
+
+                @Override
+                public void completeFuture(Exception e) { }
```
Comment (guperrot) [inline sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java]: Format code

File: sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java [hunk 4]
```diff
@@ -51,7 +54,22 @@
 
     private static final String READONLY_PARTITION_NAME = Constants.READONLY;
 
-    private static final String FAKE_TOKEN = "mock";
+    private static final TokenExchange.TokenExchangeServiceCallback TOKEN_EXCHANGE_SERVICE_CALLBACK =
```
Comment (guperrot) [inline sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java]: static final in Java can either mean static readonly or const in c# terms.
If it means static readonly then the convention is sTokenExchangeServiceCallback (an object like that is not constant compile time expression).

File: sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java [hunk 5]
```diff
@@ -51,9 +55,23 @@
 
 public class TokenTest extends AbstractStorageTest {
 
-    private static final String READONLY_PARTITION_NAME = Constants.READONLY;
+    private static final TokenExchange.TokenExchangeServiceCallback sTokenExchangeServiceCallback =
```
Comment (dhei) [inline sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java]: Love this refactoring and code re-use which makes it much more readable! 😄

**Pull request merged**